src/components/ComedyMovies.js

```diff
@@ -0,0 +1,23 @@
+import { Movie } from "./Movie"
+import { useMoviesData } from "../hooks/useMoviesData"
+import './Movies.css'
+
+export const ComedyMovies = () =>{
+    const {isLoading,data,isError,error} = useMoviesData()
+    if(isLoading){
+        return <h2>Loading...</h2>
+    }
+    if(isError){
+        return <h2>{error.message}</h2>
+    }
+    // console.log(data.data[0]['Thriller-Movies'][0].name)
+    return(
```

Comment:
You are duplicating the same logic for all genres, instead just get the genre from url params , filter out accordingly, use that to show data. Use antd "Card" to show each card of movie